ipam/allocator.go

```diff
@@ -887,6 +887,8 @@ func (alloc *Allocator) loadPersistedData() {
 	if len(alloc.seed) != 0 {
 		alloc.infof("Initialising with supplied IPAM seed")
 		alloc.createRing(alloc.seed)
+	} else if _, ok := alloc.paxos.(*paxos.Observer); ok {
+		alloc.infof("Observer - awaiting IPAM data from another peer")
```

Comment:
Can we have this message in the same format as the two, e.g. 'Initialising via observation'?